sdk/python/feast/online_response.py

```diff
@@ -66,3 +66,28 @@ def to_df(self) -> pd.DataFrame:
         """
 
         return pd.DataFrame(self.to_dict())
+
+    def event_timestamps_dict(self) -> Dict[str, List[int]]:
+        """
+        Converts GetOnlineFeaturesResponse feature event timestamps into a dictionary form.
+        Includes the entity id
+        """
+        response: Dict[str, List[int]] = {}
+
+        for result in self.proto.results:
+            for idx, feature_ref in enumerate(self.proto.metadata.feature_names.val):
+                # obtain the entity id
```

Comment:
This seems to rely on an implementation detail (that isn't enforced by the proto schema itself) that we have entities first and then the feature values (in https://github.com/feast-dev/feast/blob/master/sdk/python/feast/feature_store.py#L1269-L1269)

On top of that, I think this becomes more broken when you have multiple join keys? Seems like ideally you'd actually somehow check the feature_ref and see if it's an entity.

Alternatively, I'd just output both timestamp + values for each. That would make this simpler and not make assumptions.